backend/ringv2/interface_test.go

```diff
@@ -0,0 +1,88 @@
+package ringv2
+
+import "testing"
+
+// interface assertion
+var _ Interface = (*Ring)(nil)
```

Comment:
I'm not sure what this is for.

Reply:
It's a compile time check that `*Ring` is an `Interface`.